src/.vuepress/enhanceApp.js

```diff
@@ -56,26 +56,29 @@ export default ({ Vue, router, options }) => {
       VChip,
     },
     theme: {
-      primary: "#1263CC"
-    }
+      primary: "#1263CC",
+    },
   });
 
   var redirect_dict = {
-    '/#export' :'/core/software/pupil-player/#export',
-    '/#surface-tracking' : '/core/software/pupil-capture/#surface-tracking',
-    '/#analysis-plugins' : '/core/software/pupil-player/#analysis-plugins',
-    '/#pupil-capture' :'/core/software/pupil-capture/',
-    '/#pupil-player' : '/core/software/pupil-player/',
-    '/#network-plugins' : '/core/software/pupil-capture/#network-plugins',
-    '/#interprocess-and-network-communication' : '/developer/core/network-api/',
-    '/#annotations' : '/core/software/pupil-capture/#annotations'
-  }
+    "/#export": "/core/software/pupil-player/#export",
+    "/#surface-tracking": "/core/software/pupil-capture/#surface-tracking",
+    "/#analysis-plugins": "/core/software/pupil-player/#analysis-plugins",
+    "/#pupil-capture": "/core/software/pupil-capture/",
+    "/#pupil-player": "/core/software/pupil-player/",
+    "/#network-plugins": "/core/software/pupil-capture/#network-plugins",
+    "/#interprocess-and-network-communication": "/developer/core/network-api/",
+    "/#annotations": "/core/software/pupil-capture/#annotations",
+    "/cloud/enrichments/#reference-image-mapper":
```

Comment:
@papr there are 2 new redirects here, so should I include yours or add these to your PR instead?

Reply:
Please move the redirects from #497 to yours. Then I can dedicate #497 to the software download links.

Reply:
Shouldn't "/cloud/enrichments/#reference-image-mapper": redirect to "/invisible/explainers/enrichments/#reference-image-mapper" ? 

And if #550 comes alive, then to "/invisible/explainers/enrichments/reference-image-mapper"

Reply:
@nathakits #550 has been merged just now, so the RIM link should be updated now to /invisible/explainers/enrichments/reference-image-mapper